notebooks/Tutorial3_Embedded_processors.py

```diff
@@ -199,20 +204,26 @@
 job = client.jobs.get_avatarization_job(id=job.id, timeout=1000)
 
 # Download the avatars as a pandas dataframe
-avatars = client.pandas_integration.download_dataframe(job.result.avatars_dataset.id)
-avatars = processor.postprocess(df, avatars)
+avatars_categorical = client.pandas_integration.download_dataframe(
+    job.result.avatars_dataset.id
+)
+avatars_categorical = processor.postprocess(df, avatars_categorical)
 # -
 
-print("Number of distinct values in avatars:", avatars["Clump_Thickness"].nunique())
-avatars["Clump_Thickness"].hist()
+print(
+    "Number of distinct values in avatars:",
+    avatars_categorical["Clump_Thickness"].nunique(),
+)
+avatars_categorical["Clump_Thickness"].hist()
 
-# We observe that transforming some numeric variables to categorical can be beneficial. In our example, we preserve the proportion of each unique value where it may not be the case if we keep the variables as numeric.
+# We observe that transforming some numeric variables to categorical can be beneficial. In our example, we preserve the distribution of the variable where it may not be the case if we keep the variable as numeric.
 
 # ## Categorical variables with large cardinality
 
-# The anonymization of datasets containing categorical variables with large cardinality is not trivial and we recommend to exclude the variable from the avatarization before re-assigning it probabilistically.
+# The anonymization of datasets containing categorical variables with large cardinality is not trivial. We recommend to exclude the variable from the avatarization before re-assigning it by individual similarity (`coordinate_similarity`) or by the original row order (`row_order`) which is very **SENSITIVE** and a **violation of privacy**.
```

Comment:
do we need to mention row_order ? 
If so it should be more nuanced because we could use row_order and still have anonymous data (for example on categorical variables without a huge number of modalities). 

```suggestion
# The anonymization of datasets containing categorical variables with large cardinality is not trivial. We recommend to exclude the variable from the avatarization before re-assigning it by individual similarity (`coordinate_similarity`) or by the original row order (`row_order`). Using row order is more likely to preserve identifying information than coordinate similarity. Privacy metrics must be calculated at the end of the process to confirm that the data generated is anonymous.```